package.json

```diff
@@ -582,6 +582,7 @@
         "mocha": "^10.1.0",
         "mocha-junit-reporter": "^1.23.1",
         "mocha-multi-reporters": "^1.1.7",
+        "source-map-support": "^0.5.21",
```

Comment:
Can this dev dependency also be removed?

Reply:
I was using it in the tests to make the test errors link to .ts files instead of .js files. Without this package I couldn't get that working. I can add it in a future PR.

Reply:
Removed it. Looks like I wasn't using it anymore